config/cable.yml

```diff
@@ -1,8 +1,10 @@
 development:
-  adapter: async
+  adapter: redis
```

Comment:
we are introducing redis as a dependency in CE

what are the options? I would prefer to skip this

Reply:
@dormi no way around this unfortunately. we need `redis` for `ActionCable`.

Reply:
@aapomm https://edgeguides.rubyonrails.org/action_cable_overview.html#subscription-adapter